packages/eslint-config-4catalyzer-react/rules.js

```diff
@@ -46,4 +46,42 @@ module.exports = {
   // JavaScript.
   'react/require-default-props': 'off',
   'react/destructuring-assignment': 'off',
+  'react/sort-comp': [
+    'error',
+    {
+      order: [
+        'static-methods',
+        'type-annotations',
```

Comment:
shouldn't these go before statics? like `props` before `contextTypes` or whatever. but also, do we even use this? it looks like it's this syntax that we're not using? https://github.com/yannickcr/eslint-plugin-react/blob/aef3ee9dede3afbd5859d3130397f860202d5ab4/tests/lib/rules/sort-comp.js#L576-L598

Reply:
we use it for ts/flow how else would you define the types for instance methods? I'm happy to stick them before static methods, i was opting to stick them next to the other instance methods since they look similar

Reply:
oops i misunderstood. this is fine then.